test/customers-api.spec.js

```diff
@@ -80,6 +80,54 @@ describe('Customers API', function () {
     })
   })
 
+
+  describe('scroll', async function () {
+    it('should scroll customers descending', async function () {
+      var server = nock('https://api.voucherify.io', reqWithoutBody)
+        .get('/v1/customers')
+        .query({ scroll: true, filters: 'value' })
+        .reply(200, {has_more: true, customers: [{created_at: '2020-01-01T00:00:00Z'}, {created_at: '2020-01-02T00:00:00Z'}]})
+
+        .get('/v1/customers')
+        .query({ scroll: true, filters: 'value', created_after: '2020-01-02T00:00:00Z' })
+        .reply(200, {has_more: false, customers: [{created_at: '2020-01-03T00:00:00Z'}]})
+
+
+      let callCount = 0
+      for await (const customer of client.customers.scroll({ filters: 'value' })) {
+        ++callCount
+      }
+
+      expect(callCount).to.equal(3)
+      await server.done()
+    })
+
+
+    it('should scroll customers ascending', async function () {
+      let now = new Date().toISOString()
+      var server = nock('https://api.voucherify.io', reqWithoutBody)
+        .get('/v1/customers')
+        .query({ scroll: true, created_before: now, filters: 'value' })
+        .reply(200, {has_more: true, customers: [
+          {created_at: '2020-01-04T00:00:00Z'},
+          {created_at: '2020-01-03T00:00:00Z'},
+          {created_at: '2020-01-02T00:00:00Z'}
```

Comment:
dlaczego 4->3->2 to jest ascending? dla mnie to jest descending jak cos maleje
mam jakis dysonans poznawczy, chyba na odwrot to powinno sie nazywac w tej libce (te ify tez mi robią mindfuck-a ze uzywając `created_after` masz desceding, spodziewalbym sie na odwrot)

Reply:
if you use only `ending_before` you MUST change order, otherwise scrolling won't change.